docs/connect.html

```diff
@@ -103,7 +103,7 @@ <h4><a id="connect_configuring" href="#connect_configuring">Configuring Connecto
         <li><code>topics.regex</code> - A Java regular expression of topics to use as input for this connector</li>
     </ul>
 
-    <p>For any other options, you should consult the documentation for the connector.</p>
+    <p>For any other options, you should consult the <a href="#connectconfigs">documentation</a> for the connector.</p>
```

Comment:
That's not accurate. The documentation for the connector is not the same as the Worker configs. 
This indeed refers to the docs of each individual connector

Reply:
Yes, you're right. I read again, and they indeed refer to the docs of each individual connectors. I'll revert this change back. Thanks.